### ISSUE TITLE
Split Bill - First created bill description is not empty

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#20505]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Issue found when executing PR** https://github.com/Expensify/App/pull/20115

## Action Performed:
1. Create a split bill
2. Add All necessary info and description.
3. Create split bill.
4. Create another split bill.
5. Open the previously created bill.
6. Description and currency is as expected.
7. Create a split bill with empty description and also change the currency.
8. Create a new split bill with some description.

## Expected Result:
Open the first created split bill, description should be empty and currency should be the same that when created.

## Actual Result:
Split bill description is not empty but currency is correct

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android / native
- [ ] Android / Chrome
- [ ] iOS / native
- [ ] iOS / Safari
- [ ] MacOS / Chrome / Safari
- [ ] MacOS / Desktop

**Version Number:** 1.3.26.1

**Reproducible in staging?:** yes

**Reproducible in production?:** yes

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Notes/Photos/Videos:** Any additional supporting documentation

https://github.com/Expensify/App/assets/93399543/6db46e7b-406b-45e8-b8c7-8655238c1643

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - internal Team

**Slack conversation:**

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by joekaufmanexpensify** [comment]
I agree. i think this is expected behavior. In [the other issue](https://github.com/Expensify/App/issues/19870), the bug was that we **were** overwriting description in an existing IOU, when creating another one. 

Here the bug report seems to be that we are not doing this. But that's not a bug. When a new bill split is created, we should not overwrite the description of an existing bill split. 

@Julesssss mind sense checking my thinking here as you worked on the other issue? IMO, we should close this.

**Comment by Julesssss** [comment]
Yeah, I'm also confused by this report. It seems to be working correctly to me (but @kbecciv are we missing something?)

I'm going to close it in the meantime
